AutomationHelpers/src/UserCodeCollections/Utils.cs

```diff
@@ -26,5 +26,14 @@ public static void ReportException(Exception exception, string source)
                 exception.GetFullMessage(),
                 new SimpleReportMetadata("stacktrace", exception.StackTrace));
         }
+
+        public static string CreateRelativePath(string source, string target)
+        {
+            var u = new Uri(source);
+            var otherU = new Uri(target);
```

Comment:
Why not call these sourceUri and targetUri (why the different naming from the params)?

Reply:
Should be fixed now! :))